mungegithub/mungers/approvers/owners.go

```diff
@@ -192,9 +190,26 @@ func removeSubdirs(dirList []string) sets.String {
 	return finalSet
 }
 
+// Approval has the information about each approval on a PR
+type Approval struct {
+	Name      string // Name of the approver
+	Reason    string // How did the approver approved
```

Comment:
Nit: How did the approver approve? .  Also, what goes here?  Aren't there a limited number of ways and approver can approve...shouldn't it be more constrained that a string.  E.g. (IsAuthor, Wrote /approve, Wrote /lgtm)